ngraph_onnx/onnx_importer/ops_bridge.py

```diff
@@ -411,22 +411,22 @@ def Not(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> Ngrap
 # Variadic Ops
 def Sum(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> NgraphNode
     """Calculate element-wise sum of the input tensors."""
-    return reduce(ng.add, ng_inputs)
+    return reduce(ng.add, ng_inputs, ng.constant(0, get_dtype(ng_inputs[0].get_element_type())))
```

Comment:
Let's give this constant a name and call it by that name to document why we need it 
`ng.constant(0, get_dtype(ng_inputs[0].get_element_type()))`